javascript/javascript2/week1/hyfBayHelpers/main.js

```diff
@@ -0,0 +1,33 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const ul = document.querySelector('ul');
+
+function renderProducts(availableProducts) {
+  for(let i = 0; i < products.length ; i++){
```

Comment:
A bit verbose, you can use `availableProducts.foreach(product=>{ .... })`

Reply:
Yes I changed it to foreach.Thanks